modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -29,12 +29,14 @@ def validate_veteran_identifiers
           raise ::Common::Exceptions::UnprocessableEntity.new(detail: 'No birls_id while participant_id present')
         end
 
-        return unless header_request?
-        return if target_veteran.mpi_record?
+        if header_request? && !target_veteran.mpi_record?
+          raise ::Common::Exceptions::UnprocessableEntity.new(
+            detail: 'Submitting an original claim requires Veteran to be authenticated with an ID.me account'
```

Comment:
*requires the Veteran.

also, I'm not sure what the convention is elsewhere in `vets-api`, but I don't think veteran needs to be capitalized

Reply:
it would appear by at least the [VA.gov style guide](https://design.va.gov/content-style-guide/word-list#v) that Veteran is always capitalized, so disregard that